src/viam/sdk/common/utils.cpp

```diff
@@ -58,16 +58,14 @@ std::string bytes_to_string(const std::vector<unsigned char>& b) {
     return img_string;
 };
 
-std::chrono::time_point<long long, std::chrono::nanoseconds> timestamp_to_time_pt(
-    const google::protobuf::Timestamp& timestamp) {
+time_pt timestamp_to_time_pt(const google::protobuf::Timestamp& timestamp) {
     const std::chrono::seconds seconds(timestamp.seconds());
     const std::chrono::nanoseconds nanos(timestamp.nanos());
-    return std::chrono::time_point<long long, std::chrono::nanoseconds>(
-        std::chrono::duration_cast<std::chrono::system_clock::duration>(seconds) + nanos);
+    return time_pt(std::chrono::duration_cast<std::chrono::system_clock::duration>(seconds) +
```

Comment:
Not super familiar with the std::chrono stuff. I'm guessing a time_pt takes in a 64-bit number in the unit of nanoseconds?

Reply:
Yep! It's just a `typedef` to avoid having to write that out a bunch, details in `utils.hpp`.